packages/mui-styles/package.json

```diff
@@ -40,6 +40,7 @@
   "dependencies": {
     "@babel/runtime": "^7.23.6",
     "@emotion/hash": "^0.9.1",
+    "@mui/material": "^5.15.2",
```

Comment:
This change isn't strictly related to this PR, but I noticed this package imports @mui/material in production code, but doesn't declare it as a production dependency.

Reply:
This surprises me a bit. Intuitively I'd expect it to be the other way around. @mui/material depending on @mui/system.